src/test_utils.py

```diff
@@ -0,0 +1,63 @@
+import sys
```

Comment:
test_utils is a catch all file. Find another one which must have a meaning, maybe separate in several files.